package.json

```diff
@@ -42,6 +42,6 @@
   },
   "packageManager": "pnpm@8.14.0",
   "engines": {
-    "node": "20.x"
+    "node": "22.x"
```

Comment:
I'm upgrading the `engines` in package.json. I don't know if this is contentious or not.

Reply:
As it turns out, we should definitely be doing this. This is now our source of truth.